.travis.yml

```diff
@@ -102,7 +102,7 @@ jobs:
       script:
         - scripts/skip-doc-change.sh || travis_terminate 0;
         - make image-cephcsi || travis_terminate 1;
-        - scripts/travis-functest.sh v1.18.5 --test-cephfs=true
+        - scripts/travis-functest.sh v1.19.0 --test-cephfs=true
```

Comment:
I just have a minor question, Do we always want to use the latest release version in this job? If yes, should we directly fetch it to avoid manual updates in the future?

Reply:
not always it's better to test with the latest version. and these jobs will  be moved to centos ci and we don't need it here 
PR's are always welcome to make work simplier

Reply:
Okay, Thanks.